i2c/mpu6050_i2c/mpu6050_i2c.c

```diff
@@ -70,23 +70,29 @@ static void mpu6050_read_raw(int16_t accel[3], int16_t gyro[3], int16_t *temp) {
     // Now temperature from reg 0x41 for 2 bytes
     // The register is auto incrementing on each read
     val = 0x41;
-    i2c_write_blocking(I2C_PORT, addr, &val, 1, true);
-    i2c_read_blocking(I2C_PORT, addr, buffer, 2, false);  // False - finished with bus
+    i2c_write_blocking(i2c_default, addr, &val, 1, true);
+    i2c_read_blocking(i2c_default, addr, buffer, 2, false);  // False - finished with bus
 
     *temp = buffer[0] << 8 | buffer[1];
 }
+#endif
 
 int main() {
     stdio_init_all();
-
+#if !defined(i2c_default) || !defined(PICO_DEFAULT_I2C_SDA_PIN) || !defined(PICO_DEFAULT_I2C_SCL_PIN)
+    #warning i2c/bus_scan example requires a board with I2C pins
```

Comment:
The example here isn't i2c/bus_scan. Perhaps have a macro for the example name? Not big deal so happy to approve anyway.

Reply:
I can submit a followup fix PR

Reply:
Fixed in #75